frontend/src/components/Issue/BatchIssueActions.vue

```diff
@@ -0,0 +1,309 @@
+    tooltip-mode="DISABLED-ONLY"
+    @click="startBatchIssueTransition('REOPEN')"
+  >
+    {{ $t("issue.batch-transition.reopen") }}
+    <template #tooltip>
+      <div class="whitespace-nowrap">
+        {{
+          $t("issue.batch-transition.not-allowed-tips", {
+            operation: $t("issue.batch-transition.reopened"),
+          })
+        }}
+      </div>
+    </template>
+  </BBTooltipButton>
+
+  <BBModal
+    v-if="state.modal.show"
+    :title="state.modal.title"
+    class="relative overflow-hidden"
+    @close="state.modal.show = false"
+  >
+    <div
+      v-if="state.isRequesting"
+      class="absolute inset-0 flex flex-col items-center justify-center bg-white/70"
+    >
+      <BBSpin />
+      <div class="flex items-center textlabel">
+        <span>{{ $t("common.updating") }}</span>
+        <span v-if="state.stats"
+          >({{ state.stats.success }} / {{ state.stats.total }})</span
+        >
+      </div>
+    </div>
+
+    <div>
+      <div class="sm:col-span-4 w-112 min-w-full">
+        <label for="about" class="textlabel">
+          {{ $t("issue.status-transition.form.note") }}
+        </label>
+        <div class="mt-1">
+          <textarea
+            ref="commentTextArea"
+            v-model="state.modal.comment"
+            rows="3"
+            class="textarea block w-full resize-none mt-1 text-sm text-control rounded-md whitespace-pre-wrap"
+            :placeholder="$t('issue.status-transition.form.placeholder')"
+            @input="
+              (e) => {
+                sizeToFit(e.target as HTMLTextAreaElement);
+              }
+            "
+            @focus="
+              (e) => {
+                sizeToFit(e.target as HTMLTextAreaElement);
+              }
+            "
+          ></textarea>
+        </div>
+      </div>
+    </div>
+
+    <!-- Update button group -->
+    <div class="flex justify-end items-center pt-5">
+      <button
+        type="button"
+        class="btn-normal mt-3 px-4 py-2 sm:mt-0 sm:w-auto"
+        @click.prevent="state.modal.show = false"
+      >
+        {{ $t("common.cancel") }}
+      </button>
+      <button
+        type="button"
+        class="ml-3 px-4 py-2"
+        :class="state.modal.okButtonClass"
+        @click="doBatchIssueTransition(state.modal.transition!)"
+      >
+        {{ state.modal.okButtonText }}
+      </button>
+    </div>
+  </BBModal>
+</template>
+
+<script lang="ts" setup>
+import { computed, PropType, reactive } from "vue";
+
+import type {
+  Issue,
+  IssueStatusPatch,
+  IssueStatusTransition,
+  IssueStatusTransitionType,
+} from "@/types";
+import {
+  ASSIGNEE_APPLICABLE_ACTION_LIST,
+  CREATOR_APPLICABLE_ACTION_LIST,
+  ISSUE_STATUS_TRANSITION_LIST,
+  SYSTEM_BOT_ID,
+} from "@/types";
+import { allTaskList, hasWorkspacePermission } from "@/utils";
+import {
+  pushNotification,
+  refreshIssueList,
+  useCurrentUser,
+  useIssueStore,
+} from "@/store";
+import { useI18n } from "vue-i18n";
```

Comment:
Update import order.

```suggestion
import { computed, PropType, reactive } from "vue";
import { useI18n } from "vue-i18n";
import type {
  Issue,
  IssueStatusPatch,
  IssueStatusTransition,
  IssueStatusTransitionType,
} from "@/types";
import {
  ASSIGNEE_APPLICABLE_ACTION_LIST,
  CREATOR_APPLICABLE_ACTION_LIST,
  ISSUE_STATUS_TRANSITION_LIST,
  SYSTEM_BOT_ID,
} from "@/types";
import { allTaskList, hasWorkspacePermission } from "@/utils";
import {
  pushNotification,
  refreshIssueList,
  useCurrentUser,
  useIssueStore,
} from "@/store";
```